### ISSUE TITLE
Github Release and tag is not created when merging the release PR using the Github CLI

### ISSUE THREAD

**Issue by nils1k** [googleapis/release-please#2457]
#### Describe the error
Neither a tag, nor a release, is being created when merging the release PR using the Github CLI.

#### Steps to reproduce

  1. Push a commit so that a release PR is generated
  2. Merge the PR manually so that a release and tag are created
  3. Push a commit so that a new release PR is generated
  4. Use the Github CLI to merge the release PR, e.g. `gh pr merge <pr_number> --squash`
  5. The release PR is merged, but neither a release, nor a tag, is a created

Please refer to the [sample code](https://github.com/nils1k/release-please) that reproduces the issue.

#### Workaround
A workaround seems to be to add the `release-please:force-run` label to the merged PR, as suggested in the [README](https://github.com/googleapis/release-please?tab=readme-ov-file#step-3-rerun-release-please).

#### Notes
This issue was originally posted in the [release-please-action](https://github.com/googleapis/release-please-action/issues/1067) repository, but I'm posting it here as well as I'm unsure about where the issue belongs.

**Comment by preet248** [comment]
Yes, I am facing the same issue. But release and tags are getting created on rerunning the yml again. I am using .yml to create changeling on creating any branch name contains "Release"
**changelog.yml** 

name: Release Please

on:
  push:
    branches:
       - '*[rR][eE][lL][eE][aA][sS][eE]*' 
      
permissions:
  contents: write
  pull-requests: write

jobs:
  release-please:
    runs-on: ubuntu-latest 
    outputs:
      tag_name: ${{ steps.release.outputs.tag_name}}
      release_created: ${{ steps.release.outputs.release_created}}
    steps:
      - uses: actions/checkout@v4

      - name: Set up Node.js
        uses: actions/setup-node@v3
        with:
          node-version: '20.16.0'    

      - name: Install jq
        run: sudo apt-get install jq 

      - name: Extract npm version from package.json    
        id: extract-npm-version
        run: echo "NPM_VERSION=$(jq -r '.engines.npm' package.json)" >> $GITHUB_ENV
 
      - name: Install specific npm version
        run: npm install -g npm@${{ env.NPM_VERSION }}  
 
      - name: Run Release Please
        uses: googleapis/release-please-action@v4
        id: release
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          config-file: release-please-config.json
          manifest-file: .release-please-manifest.json
          target-branch: ${{ github.ref_name }}
  
      - name: Tag major and minor versions
        if: ${{ steps.release.outputs.release_created }}
        continue-on-error: true
        run: |
          git config user.name github-actions[bot] 
          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
          git remote add gh-token "https://${{ secrets.GITHUB_TOKEN }}@github.com/googleapis/release-please-action.git"
          git tag -d v${{ steps.release.outputs.major }} || true
          git tag -d v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
          git push origin :v${{ steps.release.outputs.major }} || true
          git push origin :v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
          git tag -a v${{ steps.release.outputs.major }} -m "Release v${{ steps.release.outputs.major }}"
          git tag -a v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} -m "Release v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}"
          git push origin v${{ steps.release.outputs.major }}
          git push origin v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}

      - name: Debug Release Outputs
        run: |
           echo "release_created: ${{ steps.release.outputs.release_created }}"
           echo "tag_name: ${{ steps.release.outputs.tag_name }}"

      # - name: Tag Release
      #   if: ${{ steps.release.outputs.release_created == 'true' }}
      #   run: |
      #     git config user.name "github-actions[bot]"
      #     git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
      #     git tag ${{ steps.release.outputs.tag_name }}
      #     git push origin ${{ steps.release.outputs.tag_name }}    
      #
 
      - name: Print release outputs for debugging
        continue-on-error: true
        run: |
          echo "Release created: ${{ steps.release.outputs['release_created'] }}"
          echo "Tag name: ${{ steps.release.outputs['tag_name'] }}"
          echo "Release outputs:"
          echo "${{ steps.release.outputs }}"
          echo "${{ toJson(steps.release.outputs) }}"
 
      - name: Create CHANGELOG.md
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        run: |
          if [[ -f CHANGELOG.md ]]; then
            echo "Appending changelog..."
          else
            echo "Creating changelog..."
            echo "# Changelog" > CHANGELOG.md
          fi
          gh release view --json body --jq .body | tee -a CHANGELOG.md

      - name: Delete Git Tags
        if: success()
        run: |
               git fetch --tags
                 TAGS=$(git tag)
                for TAG in $TAGS; do
                 echo "Deleting tag $TAG"
                  git tag -d $TAG   
                 git push origin :refs/tags/$TAG  
                 done
      

      - name: Auto-Merge Release PR
        env:
            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        run: |
            PR_NUMBER=$(gh pr list --state open --json number --jq '.[0].number')

            echo "PR Number: $PR_NUMBER"
            gh pr merge $PR_NUMBER --merge

**.release-please-manifest.json**
 
{
    ".": "1.6.0"
}

release-please-config.json
{
    "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
    "tag-separator": "@",
    "separate-pull-requests": true,
    "packages": {
      ".": {
        "package-name": "changelog25",
        "release-type": "node"
      }
    }
  } 

I am trying to do this workaround

"A workaround seems to be to add the release-please:force-run label to the merged PR, as suggested in the [README](https://github.com/googleapis/release-please?tab=readme-ov-file#step-3-rerun-release-please)"

But how to do it ?Could anyone help?
